images/macos/templates/macOS-11.pkr.hcl

```
@@ -0,0 +1,220 @@
+variable "source_vm_name" {
+  type = string 
+  default = "clean_macos_11_300gb"
+}
+
+variable "source_vm_tag" {
+  type = string
+  default = "bigsur_300gb"
+}
+
+variable "vm_name" {
+  type = string
+  default = "macos-11"
+}
+
+variable "vcpu_count" {
+  type = string
+  default = "4"
+}
+
+variable "ram_size" {
+  type = string
+  default = "6G"
+}
+
+variable "build_id" {
+  type = string
+}
+
+variable "vm_username" {
+  type = string
+}
+
+variable "vm_password" {
+  type = string
+}
+
+variable "xcode_install_user" { type = string }
+variable "xcode_install_password" { type = string }
+
+variable "image_os" {
+  type = string
+  default = "macos11"
+}
+         
+source "veertu-anka-vm-clone" "template" {     
+  vm_name = "${var.vm_name}"      
+  source_vm_name = "${var.source_vm_name}"    
+  source_vm_tag = "${var.source_vm_tag}"      
+  vcpu_count = "${var.vcpu_count}"
+  ram_size = "${var.ram_size}"    
+  stop_vm =  "true"
+}        
+         
+build {  
+  sources = [        
+    "source.veertu-anka-vm-clone.template",       
```

```suggestion
source "veertu-anka-vm-clone" "template" {
  vm_name = "${var.vm_name}"
  source_vm_name = "${var.source_vm_name}"
  source_vm_tag = "${var.source_vm_tag}"
  vcpu_count = "${var.vcpu_count}"
  ram_size = "${var.ram_size}"
  stop_vm =  "true"
}

build {
  sources = [
    "source.veertu-anka-vm-clone.template",
```

Remove trailing spaces